src/RoleDB.php

```diff
@@ -68,7 +69,7 @@ public function __construct()
     public function addRole(Role $role): void
     {
         if ($this->roleExists($role->getName())) {
-            throw new \Exception('Role already exists: ' . $role->getName());
+            throw new RoleExistsException('Role already exists: ' . $role->getName());
```

Comment:
Looks like we don't have any existing test coverage for RoleDB, which is a shame. Can you open a follow-up issue (no need for a PR yet) to remind us to add test coverage for RoleDB?

Reply:
Filed #178 to address this. :)